backend/tests/conftest.py

```
@@ -0,0 +1,83 @@
+# -------------------------------------------------
+
+class FakeMCQGenerator:
+    def generate_mcq(self, *args, **kwargs):
+        return {"questions": []}
+
+class FakeFileProcessor:
+    def process(self, *args, **kwargs):
+        return ""
+
+class FakeShortQGenerator:
+    def generate_shortq(self, *args, **kwargs):
```

Ruff (0.15.2)

[warning] Unused method argument: `args`

(ARG002)